python/test/t_SymbolicFunction_std.py

```
@@ -257,3 +257,13 @@
 print(f.__repr__())
 print("HTML:")
 print(f._repr_html_())
+
+# crash in expression initialization
+f = ot.SymbolicFunction(["x0"], ["zou * 42"])
+x = ot.Uniform().getSample(10)
+for i in range(5):
```

Ruff (0.15.18)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)